gulpfile.js

```diff
@@ -97,6 +97,13 @@ const copyGtmHead = () => {
     .pipe(dest(paths.dist + 'js/'));
 };
 
+// Task to copy `setTimezone.js`
+const copySetTimezone = () => {
```

Comment:
Do you need to add this to reference the js? Or could you add to the list on line 82? Is it because you don't want this to compile to the all.js  javascript?

Reply:
I don't want it to compile to all.js, because then it would be invoked everywhere, all the time.  I only want it to invoke on a page load of a page everyone hits sooner or later.